src/pages/RepoPage/SettingsTab/tabs/BadgesAndGraphsTab/Badges/Badges.tsx

```diff
@@ -0,0 +1,150 @@
+import PropTypes from 'prop-types'
+import { useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import config from 'config'
+
+import { Branch, useBranches } from 'services/branches'
+import A from 'ui/A'
+import Select from 'ui/Select'
+import SettingsDescriptor from 'ui/SettingsDescriptor'
+import TokenWrapper from 'ui/TokenWrapper'
+
+type UseBadgesProps = {
```

Comment:
`l`: I would swap `Props` for `Args` personally, as this isn't a component and just a function (hook), that way people won't try and use it or confuse it with a component.

Reply:
This is interesting. Didn't know the terminology worked like that TY!

Reply:
There is actually an `eslint` rule for it somewhere

Reply:
This is more for clarity, it's not gonna end the world or anything ... BUT when I see the word props I immediately assume it's for a component